gpjax/config.py

```diff
@@ -48,6 +52,8 @@ def get_defaults() -> ConfigDict:
     transformations.triangular_transform = FillTriangular
 
     # Default parameter transforms
+    transformations.alpha = "positive_transform"
+    transformations.bias = "positive_transform"
```

Comment:
I would be careful here. To me, the naming "bias", gets me thinking of an element of $\mathbb{R}$ not $\mathbb{R}_{+}$.